components/bitly/actions/expand-bitlink/expand-bitlink.mjs

```diff
@@ -0,0 +1,38 @@
+import { axios } from "@pipedream/platform";
+import { ConfigurationError } from "@pipedream/platform";
+
+export default {
+  key: "bitly-expand-bitlink",
+  name: "Expand a Bitlink",
+  description: "Retrieves information about Bitlink using ID.",
+  version: "0.0.1",
+  type: "action",
+  props: {
+    bitly: {
+      type: "app",
+      app: "bitly",
+    },
+    bitlink_id: {
+      type: "string",
+      description: "Bitlink ID",
+    },
+  },
+  async run({ $ }) {
+    const { bitlink_id } = this;
+    const data = { bitlink_id };
+
+    try {
+      return await axios($, {
+        method: "post",
+        url: "https://api-ssl.bitly.com/v4/expand",
+        headers: {
+          Authorization: `Bearer ${this.bitly.$auth.oauth_access_token}`,
+          "Content-Type": "application/json",
+        },
+        data,
+      });
+    } catch (error) {
```

Comment:
I think you can just throw the error to the user, so it can check the reason for the error. So I would like to suggest you to remove this try/catch

Reply:
Done